<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Detail</class>
 <widget class="QDialog" name="Detail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>20</width>
     <height>461</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="city_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>160</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QLabel" name="time_1">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>62</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>t1</string>
   </property>
  </widget>
  <widget class="QLabel" name="time_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>122</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>t2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="city_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>160</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>B</string>
   </property>
  </widget>
  <widget class="QPushButton" name="city_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>160</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QLabel" name="time_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>181</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>t3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="city_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>160</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>D</string>
   </property>
  </widget>
  <widget class="QLabel" name="time_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>t4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="city_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>160</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>E</string>
   </property>
  </widget>
  <widget class="QLabel" name="time_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>t5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="city_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>160</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>F</string>
   </property>
  </widget>
  <widget class="QLabel" name="time_6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>370</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>t6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="city_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>160</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>G</string>
   </property>
  </widget>
  <widget class="QLabel" name="time_7">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>430</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>t7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pgup">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>470</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>上一页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pgdn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>470</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string> 下一页</string>
   </property>
  </widget>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>301</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>45</pointsize>
    </font>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>50</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="text">
    <string>上次更新时间：</string>
   </property>
  </widget>
  <widget class="QLabel" name="c_time">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>90</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>12:33</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>150</y>
     <width>135</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>温度       :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>220</y>
     <width>135</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PM10   ：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>290</y>
     <width>135</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PM2.5  ：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>360</y>
     <width>135</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SO2      ：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>430</y>
     <width>135</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>空气质量：</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="temp">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>150</y>
     <width>380</width>
     <height>40</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="pm10">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>220</y>
     <width>380</width>
     <height>40</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="pm25">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>290</y>
     <width>380</width>
     <height>40</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="so2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>360</y>
     <width>380</width>
     <height>40</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="quality">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>430</y>
     <width>380</width>
     <height>40</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>490</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="temp_v">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>150</y>
     <width>71</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QLabel" name="pm10_v">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>220</y>
     <width>71</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QLabel" name="pm25_v">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>290</y>
     <width>71</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QLabel" name="so2_v">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>360</y>
     <width>71</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QLabel" name="quality_v">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>430</y>
     <width>71</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
